packages/elements/src/photon-patient-form/index.tsx

```diff
@@ -58,30 +59,34 @@ customElement(
       address_city: undefined,
       address_state: undefined,
       address_zip: undefined,
-      preferredPharmacy: undefined,
+      preferredPharmacy: undefined
     });
     actions.registerValidator({
       key: 'firstName',
-      validator: message(size(string(), 1, Infinity), 'Please enter a first name...'),
+      validator: message(size(string(), 1, Infinity), 'Please enter a first name...')
     });
     actions.registerValidator({
       key: 'lastName',
-      validator: message(size(string(), 1, Infinity), 'Please enter a last name...'),
+      validator: message(size(string(), 1, Infinity), 'Please enter a last name...')
+    });
+    actions.registerValidator({
+      key: 'sex',
+      validator: message(enums(sexes.map((s) => s.name)), 'Please enter Sex at Birth...')
     });
     actions.registerValidator({
       key: 'phone',
-      validator: message(size(string(), 10, 12), 'Please enter a phone number...'),
```

Comment:
Isn't 10 a minimum?

Reply:
it is, but the input converts 2223334444 (10) into +12223334444 (12)